xla/service/gpu/runtime/dynamic_slice_thunk.cc

```diff
@@ -47,14 +48,30 @@ limitations under the License.
 namespace xla {
 namespace gpu {
 
+namespace {
+
+// Indvar is a thread-local map that stores the induction variable for each
+// dynamic slice thunk. The usage of threadlocal in this context is similar to
+// `LoopCounters` in while_thunk.cc (b/343294327).
+Literal& Indvar(DynamicSliceThunk* thunk) {
```

Comment:
> This deserves documentation.

I have added a comment explaining what it is, and also a reference to the same comment that the LoopCounters had. Hopefully that will be helpful.